packages/core/core/src/requests/AssetRequest.js

```diff
@@ -111,18 +147,48 @@ async function run({input, api, options, farm, invalidateReason}: RunInput) {
     }
   }
 
+  for (let devDepRequest of devDepRequests) {
+    await api.runRequest<null, void>({
```

Comment:
Should these subrequests execute in parallel?

Reply:
Well, they don't really do anything other than add the invalidations so I'm not sure it would really be much benefit?